test/slicing/test_convergence.py

```diff
@@ -87,27 +87,27 @@ def test_convergence(self):
         model = SnorkelClassifier(tasks=tasks)
 
         # Train
-        trainer = Trainer(lr=0.01, n_epochs=30, progress_bar=False)
+        trainer = Trainer(lr=0.001, n_epochs=50, progress_bar=False)
```

Comment:
Hm I don't love fiddling with hyperparams in tests. Makes it feel brittle. Is this due to a change in this diff or just globally better settings?

Reply:
They seemed a bit globally better (they are consistent across the 2 integration tests now). Earlier I tried a higher `lr` to reach convergence more quickly for shortened tests, but I'm prioritizing tighter performance bounds (>95F1) for now—we can shorten the tests down the line if necessary.